lib/allowlist.py

```diff
@@ -33,6 +33,9 @@
     'jkonecny12',
     'rvykydal',
```

Comment:
Please add him to installer team + contributors just above. This is for the repo maintainers to know whom to contact for future cleanups.